scipy/ndimage/tests/test_measurements.py

```diff
@@ -1390,3 +1390,14 @@ def test_watershed_ift08(self):
         expected = [[1, 1],
                     [1, 1]]
         assert_array_almost_equal(out, expected)
+
+    def test_watershed_ift09(self):
+        # Test large cost. See gh-19575
+        data = np.array([[(1 << 16) - 1, 0],
```

Comment:
I wonder if it might be clearer to use the NumPy approach to getting at the largest `uint16` like

```diff
--- a/scipy/ndimage/tests/test_measurements.py
+++ b/scipy/ndimage/tests/test_measurements.py
@@ -1393,7 +1393,7 @@ class TestWatershedIft:
 
     def test_watershed_ift09(self):
         # Test large cost. See gh-19575
-        data = np.array([[(1 << 16) - 1, 0],
+        data = np.array([[np.iinfo(np.uint16).max, 0],
                          [0, 0]], np.uint16)
```

I guess it doesn't matter much, just takes longer to parse the bit shift for those of us that may do less low-level programming I guess.

Reply:
Sure!